pkg/scorecard.go

```diff
@@ -22,14 +23,13 @@ import (
 	"time"
 
 	"github.com/google/go-github/v32/github"
-	"github.com/shurcooL/githubv4"
```

Comment:
Revert changes in this file

Reply:
it seems the dependency is not needed. I did not purposely remove it, but my editor figured it was not needed and removed it when I saved the file. No harm in removing it then?